packages/common/react-native-vector-icons.podspec

```diff
@@ -81,21 +80,6 @@ Pod::Spec.new do |s|
   if respond_to?(:install_modules_dependencies, true)
     install_modules_dependencies(s)
```

Comment:
since RN 0.71, `install_modules_dependencies` is the only thing needed to integrate with RN. It replaces the deleted code also.

Reply:
@vonovak I was aware of that, but I'm unsure whether we are ready to drop support for pre 0.71.

@oblador What are your thoughts re how far back we should support?

Reply:
Hello, this doesn't drop support for pre-71.

Reply:
Sorry I misread that, thought the entire else was going away